### PR TITLE
Add support for module tag

### PR DESCRIPTION
**Pull request by tineheller**
1. The since-test failed, so I changed the part where the versionComparator is added. I'm not sure, if this is how you meant it. If not, maybe you can change it again. For now, the since-tests run and I think the versionComparator does what I'd expect :)
2. I'm still not sure about the flags. I read that a module member is exported if it's static. The probem is, if it's not exportet, there is still a declare added to the output... I'm not sure if this is correct?

### CODE REVIEW

File: src/test/module/test.parseModule.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+		let variableMember: dom.VariableDeclaration = parsedModule.members[0] as dom.VariableDeclaration;
+		expect(variableMember.name).to.eq("moduleMember");
+		expect(variableMember.jsDocComment).to.eq("my module member");
+
+		let unionType = variableMember.type as dom.UnionType;
+		expect(unionType.members.length).to.eq(1);
+		expect(unionType.members[0]).to.eq(dom.type.string);
+	});
+})
```
Comment (wehrstedt) [inline src/test/module/test.parseModule.ts]: missing semicolon and new line at the end

File: package.json [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
+	"dependencies": {
+		"@types/node": "^8.0.29",
+		"@types/shelljs": "^0.7.6",
+		"dts-dom": "^0.1.22",
+		"typescript": "^2.5.2"
+	},
+	"peerDependencies": {
+		"jsdoc": "^3.5.5"
+	}
+}
```
Comment (wehrstedt) [inline package.json]: can you change the line ending back to "CRLF" (or "LF" if it was "LF" before)?

**Comment by wehrstedt** [comment]
> I'm still not sure about the flags. I read that a module member is exported if it's static. The probem is, if it's not exportet, there is still a declare added to the output... I'm not sure if this is correct?

It's definitily not correct that module members will be declared, because that's not valid typescript. But I'm not sure how to deal with that for now. Actually we should fix that in the dts-dom module...
Fact is, that every module member, that is not explicity exportet, will be private or rather the access description will be omitted.

**Comment by tineheller** [comment]
> Fact is, that every module member, that is not explicity exportet, will be private or rather the access description will be omitted.

But how is it explicitly exported in jsDoc? I tried to list the members in @exports (http://usejsdoc.org/tags-exports.html) but it didn't work. The export was not even shown in the JSON output of jsdoc...
Then, after reading that http://usejsdoc.org/tags-module.html#examples I thought, maybe static members should be exported. But yes, the members are all static on default, so they are not explicitly exported.

**Comment by tineheller** [comment]
I just realised, the line ending fix didn't work. Maybe I have to change something in my git settings...?

**Comment by wehrstedt** [comment]
In my opinion every member is public until you add the private-tag.

Possible. Search for git line ending and set it to "checkout as is, commit as is"

**Comment by tineheller** [comment]
seems it has been CRLF before... I don't really understand but hope it's correct again now

**Comment by wehrstedt** [comment]
Looks good. Maybe because of your git settings it was changed to LF. I will publish the new version tomarrow, ok?

**Pull request merged**